src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -340,6 +340,8 @@ public static LanguageVersion MapSpecifiedToEffectiveVersion(this LanguageVersio
             }
         }
 
+        internal static LanguageVersion CurrentVersion => LanguageVersion.CSharp8;
```

Comment:
I suspect we're going to add `LanguageVersion.Preview` or some such. That would do the job. Tagging @agocke 

I'm fine to add this property in the meantime, but we'll probably want to remove it once we have a symbolic label for "preview".

Reply:
Agreed